### DISCUSSION TITLE
With the 3.x redesign did the "Mark all as read" time selection get removed?

### DISCUSSION THREAD

**Discussion by matthewlenz** [Athou/commafeed#1145] [General]
I'm still running  2.x and thought I'd check out the demo on commafeed.com and noticed that the little drop down to select the time for marking read was missing.

**Comment by Athou** [comment]
The feature is still there, it moved in the confirmation popup :)

![image](https://github.com/Athou/commafeed/assets/1256795/0e2a6500-d9cd-4441-bfa0-dff5602312c8)

**Reply by matthewlenz** [reply]
Ahh ok thanks.  I didn't even think to just click it :D  Ironically the one I used the most was the 12hr option.  Maybe I'll just try to fiddle around directly in the DB to keep specific groups limited to 12hrs instead (if such a thing is possible).
